pkg/steps/template.go

```diff
@@ -490,7 +489,7 @@ func waitForPodCompletion(ctx context.Context, podClient coreclientset.PodInterf
 		case <-ctxDone:
 		default:
 			skipLogs = true
-			if !retry || err == nil {
```

Comment:
Why are we changing this?

Reply:
Because there is no retry without a watch